src/services/executeWrappers/TerminalCommandExecuter.ts

```diff
@@ -93,6 +99,34 @@ export class TerminalCommandExecuter {
     commands.executeCommand(Commands.executeTerminalCommand, `gulp serve --config=${answer}`);
   }
 
+  public static async bundleProject() {
+    const tasks = ['local','production'];
+    const answer = await window.showQuickPick(tasks, {
+      title: 'Select the target environment',
+      ignoreFocusOut: true
+    });
+
+    if(!answer) {
+      return;
+    }
+
+    commands.executeCommand(Commands.executeTerminalCommand, answer === 'local' ? 'gulp bundle' : 'gulp bundle --ship');
```

Comment:
We could simplify and make it a bit shorter like.
```suggestion
    commands.executeCommand(Commands.executeTerminalCommand, `gulp bundle${answer === 'local' ? '' : ' --ship'}`);
```

What do you think?
We could do the same for package as well